lib/passthrough.lua

```diff
@@ -134,7 +135,12 @@ function Passthrough.init()
                 core.build_scale(params:get("root_note_"..v.port), params:get("current_scale_"..v.port), v.port)
               end
             }
-    
+          params:add {
+            type = "option",
+            id = "cc_limit_"..v.port,
+            name = "CC limit",
+            options = core.cc_limits
+          }
```

Comment:
Since you've added a new parameter, line 52 needs to be changed to `params:add_group("PASSTHROUGH", 10*port_amount + 2)` so that the "All devices" separator and Midi panic toggle param are still visible in the passthrough group in the params and not outside.